runtime/doc/usr_07.txt

```diff
@@ -242,8 +242,8 @@ Now edit the file "bar.txt" and place the B mark (B for bar) at its last line:
 >
 	GmB
 
-Now you can use the "'F" command to jump back to halfway foo.txt.  Or edit yet
-another file, type "'B" and you are at the end of bar.txt again.
+Now you can use the "'F" command to jump back to halfway through foo.txt.  To
```

Comment:
'to halfway of foo.txt' ? I wouldn't ever use 'of' in like that but I'll change if it you prefer it.